components/Popover/Popover.tsx

```diff
@@ -30,28 +30,30 @@ const StyledContent = styled(PopoverPrimitive.Content, panelStyles, {
   },
 });
 
+const StyledArrow = styled(PopoverPrimitive.Arrow, {
+  fill: 'currentColor',
```

Comment:
I think here we might actually want to pick the same color from `panelStyles.backgroundColor` to avoid this behavior:

![Screen Shot 2022-03-14 at 12 24 01](https://user-images.githubusercontent.com/38889364/158204510-425ad7a9-a092-4034-a4da-6269be9f245f.png)

Reply:
I reworked `fill` colors based on `elevationVariants`. It seems to work more consistently